packages/common/src/content/_internal/hostedServiceUtils.ts

```diff
@@ -0,0 +1,75 @@
+import { IFeatureServiceDefinition } from "@esri/arcgis-rest-feature-layer";
+import { IItem } from "@esri/arcgis-rest-portal";
+import { IHubEditableContent } from "../../core/types/IHubEditableContent";
+
+export function isHostedFeatureService(
+  content: IHubEditableContent | IItem
+): boolean {
+  return (
+    content.type === "Feature Service" &&
+    content.typeKeywords.includes("Hosted Service")
+  );
+}
+
+export enum ServiceCapabilities {
+  EXTRACT = "Extract",
+}
+
+/**
+ * Returns a whether a service has a capability
+ * @param capability
+ * @param serviceDefinition
+ *
+ * @returns {boolean}
+ */
+export function hasCapability(
```

Comment:
I wonder if this should be `hasServiceCapability()` in order to avoid confusion w/ other capabilities.

Reply:
never mind. I think this is fine given the other fns.

Reply:
@tomwayson I agree with your original sentiments. I'll be a small change.